charts/hdfs-config-k8s/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: '1.0'
```

Comment:
Maybe remove this?  I'm not sure what it means in this context since it's not tied to the hadoop version that will be installed